### PR TITLE
Versioning specificity

### PR DESCRIPTION
**Pull request by tkelman**

### CODE REVIEW

**Comment by ararslan** [comment]
Looks like the OS X nightly failure is https://github.com/JuliaLang/julia/issues/17484 and the AppVeyor failure is because Rmath and Windows are mortal enemies.

Thanks for doing this, looks good to me.

**Comment by tkelman** [comment]
Rmath on Windows is next on my list, really.

File: src/abstractdataframe/io.jl [hunk 1]
```diff
@@ -8,7 +8,7 @@ function escapedprint(io::IO, x::Any, escapes::AbstractString)
     print(io, x)
 end
 
-@static if VERSION < v"0.5.0-"
+@static if VERSION < v"0.5.0-dev+4354"
```
Comment (bicycle1885) [inline src/abstractdataframe/io.jl]: Is this `@static` macro call needed at the top level?

Comment (tkelman) [inline src/abstractdataframe/io.jl]: no, probably not

**Pull request merged**